db/models/Comment.js

```diff
@@ -9,7 +9,7 @@ module.exports = (sequelize, DataTypes) => {
   Comment.associate = (models) => {
     Comment.belongsTo(models.User, {
       foreignKey: 'userId',
-      as: 'user'
```

Comment:
can see you are trying to associate the comments to the user table, Good job. since comments will be made by users and not authors wouldn't it be nicer if the comments relate to the users table with the keyword 'users' rather than 'authors'?